src/main/scala/ai/privado/exporter/ConsoleExporter.scala

```diff
@@ -0,0 +1,138 @@
+package ai.privado.exporter
+
+import ai.privado.model.exporter.{CollectionModel, DataFlowSubCategoryModel, SourceModel, SourceProcessingModel}
+
+import java.net.URL
+import scala.collection.mutable
+import Console.{BLUE, BOLD, CYAN, GREEN, MAGENTA, RED, RESET, WHITE, YELLOW}
+
+object ConsoleExporter {
+
+  def exportConsoleSummary(
+    dataflowsOutput: mutable.LinkedHashMap[String, List[DataFlowSubCategoryModel]],
+    sources: List[SourceModel],
+    processing: List[SourceProcessingModel],
+    collections: List[CollectionModel],
+    violationSize: Int
+  ): Unit = {
+    // Parse the Dataflows
+    val sourceNameIdMap = mutable.HashMap[String, String]()
+    val leakageSourceMap = mutable.HashMap[String, Int]()
+    val processSourceMap = mutable.HashMap[String, Int]()
+    val collectionsSourceMap = mutable.HashMap[String, mutable.Set[String]]()
+    val storageSourceMap = mutable.HashMap[String, mutable.Set[String]]()
+    val thirdPartySourceMap = mutable.HashMap[String, mutable.Set[String]]()
+    val internalAPIsSourceMap = mutable.HashMap[String, mutable.Set[String]]()
+    val uniqueThirdParties = mutable.Set[String]()
+
+    // SourceId - Name Map
+    sources.foreach(source => {
+      sourceNameIdMap.addOne(
+        source.id.replaceAll("\"", "") -> source.name.replaceAll("\"", "")
```

Comment:
@dbMundada Do we need this step of replace?

Reply:
Scalalish/java way of doing this can be (non blocker)

val sourceNameIdMap = sources.map(source => (source.id, source.name)).toMap